Automattic/wp-calypso
# Enable My Home for all roles

Partially fixes https://github.com/Automattic/jetpack/issues/19506.

#### Changes proposed in this Pull Request

- My Home is now enabled for Editors, Authors, and Contributors. 
- Content specifically oriented for Administrator is not visible for those users.

#### Testing instructions

- Apply D71134-code and D71149-code to your sandbox.
- Sandbox the API.
- Use the Calypso live link below.
- As an Editor, Author, or Contributor:
  - Make sure you have access to `/home/:site` (a direct access from the menu will be handled in a separate PR).
  - Make sure the content displayed is ok for non-admins.
- As an Administrator, make sure there are no regressions on My Home.
